servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/RedoPublisher.java

```diff
@@ -49,9 +49,9 @@ final class RedoPublisher<T> extends AbstractNoHandleSubscribePublisher<T> {
     void handleSubscribe(Subscriber<? super T> subscriber, ContextMap contextMap,
                          AsyncContextProvider contextProvider) {
         // For the current subscribe operation we want to use contextMap directly, but in the event a re-subscribe
-        // operation occurs we want to restore the original state of the AsyncContext map, so we save a copy upfront.
```

Comment:
the copy isn't done consistently after reloading changes were done a while back. I'm also not sure we need the copy any more so I made them consistent.

Reply:
moved to https://github.com/apple/servicetalk/pull/2670